### PR TITLE
aemelyanenko: bug fix hydration within hydration with variables

### PR DESCRIPTION
**Pull request by temaEmelyan**
wip for the bugfix

### CODE REVIEW

File: test/src/test/resources/fixtures/replicate-bug.yml [hunk 1]
```diff
@@ -0,0 +1,212 @@
+name: replicate bug
+enabled:
+  current: true
+  nextgen: false
+overallSchema:
+  #language=graphQL
```
Comment (gnawf) [inline test/src/test/resources/fixtures/replicate-bug.yml:6]: Idk about you, but this turns out really bad for me. You always get syntax errors saying you redefined a field or something rather

File: test/src/test/resources/fixtures/hydration-within-hydration-with-variables.yml [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    board {
+      issue {
+        comments(first: $first) {
+          totalCount
+        }
+      }
+    }
+  }
+variables: {
+  "id": "some-id",
```
Comment (andimarek) [inline test/src/test/resources/fixtures/hydration-within-hydration-with-variables.yml:87]: remove that variable if not used/

**Review by andimarek — CHANGES_REQUESTED** [review]
see comment

File: test/src/test/resources/fixtures/hydration-within-hydration-with-variables.yml (hunk 2 above)
Comment (temaEmelyan) [reply test/src/test/resources/fixtures/hydration-within-hydration-with-variables.yml]: done

**Review by andimarek — APPROVED** [review]

**Pull request merged**